MMA Removal Utility/SingleTenantSetupInstallation.md

```diff
@@ -0,0 +1,302 @@
+```
+
+**Parameter details:**
+|Param Name|Description|Required?
+|----|----|----|
+|SubscriptionId| Subscription id in which remediation MI needs to be created.| Yes|
+|ResourceGroupName| Resource group name in which remediation MI needs to be created. Default value is 'AzTS-MMARemovalUtility-RG'| No|
+|Location| Location in which remediation MI needs to be created. For better performance, we recommend hosting the MI and AzTS MMA Removal Utility setup in the same location. Default value is 'EastUS2'| No|
+|UserAssignedIdentityName| Name of the remediation MI.| Yes|
+|TargetSubscriptionIds| List of target subscription id(s) on which AzTS MMA Removal Utility setup will run.| No|
+|TargetManagementGroupNames| List of target management group name(s) on which AzTS MMA Removal Utility setup will run.| No|
+|TenantScope| Switch for tenant scope in case you want to scope the run at tenant level, which helps in assigning appropriate roles at the tenant scope for the tenant id from the context of user.| No|
+
+[Back to top…](#steps-to-install-single-tenant-azts-mma-removal-utility)
+
+<br/>
+
+### **Step 4 of 5. Setup installation**  
+Setup installation can be done using Install-AzTSMMARemovalUtilitySolution. This command will install the MMA Removal Utility which helps to discover and remove MMA agent installed on Virtual Machines.
+
+> _Note:_
+> _For creation of the setup in a new RG, user running this script should have **Owner** access on the subscription in which setup is being created. If the setup is being installed in an existing RG, user should have **Owner** access on the RG._
+> 
+
+Install-AzTSMMARemovalUtilitySolution PowerShell command will perform the following operations:
+
+- Prompts and collects onboarding details for usage telemetry collection based on user preference.
+- Creates the RG if it does not exist.
+- Creates or updates the resources with MIs assigned.
+- Creates or updates the monitoring dashboard.
+
+``` PowerShell
+# -----------------------------------------------------------------#
+# Setup installation
+# -----------------------------------------------------------------#
+
+$SetupInstallation = Install-AzTSMMARemovalUtilitySolution ` 
+         -SubscriptionId <HostingSubId> `
+         -HostRGName <HostingRGName> `
+         -Location <Location> `
+         -ManagedIdentityId <MIResourceId> `
+         -AzureEnvironmentName <AzureEnvironmentName> `
+         -DisableUsageTelemetry
+```
+
+**Parameter details:**
+|Param Name|Description|Required?
+|----|----|----|
+|SubscriptionId| Subscription id in which setup needs to be created.| Yes|
+|HostRGName| Resource group name in which setup needs to be created. Default value is 'AzTS-MMARemovalUtility-Host-RG'| No|
+|Location| Location in which setup needs to be created. For better performance, we recommend hosting the MI and AzTS MMA Removal Utility setup in the same location. Default value is 'EastUS2'| No|
+|ManagedIdentityId| ResourceId of the remediation MI.| Yes|
```

Comment:
can we provide steps of how they could get the MI Id from last step

Reply:
Tweaked the command to directly hold the value generated above. Also in the above step mentioned the same for which MI Resource Id is required.